spec/system/admin_plugins_list_spec.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+describe "Admin Plugins List", type: :system, js: true do
+  fab!(:current_user) { Fabricate(:admin) }
+
+  before { sign_in(current_user) }
+
+  let(:spoiler_alert_plugin) do
+    plugins = Plugin::Instance.find_all("#{Rails.root}/plugins")
+    plugins.find { |p| p.name == "spoiler-alert" }
+  end
+
+  xit "shows the list of plugins" do
```

Comment:
Ah I meant to say, I can't run this because you need `LOAD_PLUGINS=1` turned on for even core plugins to be loaded...not sure what the solution is but I still wanted the system spec to be there for now.